README.md

```diff
@@ -241,16 +241,38 @@ This service is responsible for running a short-circuit analysis and storing res
 
+### Time-series server
+
+- Kind: Web service with a REST API
+- Source repository: https://github.com/gridsuite/timeseries-server
+- Storage: PostgreSQL
+- Connected to message broker: no
+- Other services dependencies:
+- Use PowSyBl libraries: yes
+
+This service is responsible for storing time-series data. It is used to persist the output data of dynamic simulations which are displayed later in GridStudy.
+
 ### Dynamic simulation server
 
 - Kind: Web service with a REST API
 - Source repository: https://github.com/gridsuite/dynamic-simulation-server
 - Storage: PostgreSQL
 - Connected to message broker: producer and consumer
-- Other services dependencies: network store server
+- Other services dependencies: network store server, report server, dynamic mapping server, time-series-server
+- Use PowSyBl libraries: yes
+
+This service is responsible for running dynamic simulations. It is based on [PowSyBl dynamic simulation API](https://github.com/powsybl/powsybl-core/tree/main/dynamic-simulation/dynamic-simulation-api) and [Dynawo implementation](https://github.com/powsybl/powsybl-dynawo). This is still an ongoing work and it still doesn't work completely with the current version of the code. This is not yet completely integrated into GridStudy.
+
+### Dynamic security analysis server
+
+- Kind: Web service with a REST API
+- Source repository: https://github.com/gridsuite/dynamic-security-analysis-server
+- Storage: PostgreSQL
+- Connected to message broker: producer and consumer
+- Other services dependencies: network store server, report server, dynamic simulation server
 - Use PowSyBl libraries: yes
 
-This service is responsible for running dynamic simulations. It is based on [PowSyBl dynamic simulation API](https://github.com/powsybl/powsybl-core/tree/main/dynamic-simulation/dynamic-simulation-api) and [Dynawo implementation](https://github.com/powsybl/powsybl-dynawo). This is still an ongoing work and it still doesn't work completely with the current version of the code. This is not yet integrated into GridStudy.
+This service is responsible for running dynamic security analysis. It is based on [PowSyBl dynamic security analysis API](https://github.com/powsybl/powsybl-core/tree/main/dynamic-security-analysis) and [Dynawo implementation](https://github.com/powsybl/powsybl-dynawo). This is still an ongoing work and it still doesn't work completely with the current version of the code. This is not yet completely integrated into GridStudy.
```

Comment:
Same here

Reply:
DONE in https://github.com/gridsuite/documentation/pull/16/commits/c536a6b68417567e27b8c6840abbf5c4a42c09dd